pygmt/tests/test_figure.py

```diff
@@ -198,7 +198,7 @@ def test_figure_shift_origin():
     """
     Test if fig.shift_origin works.
     """
-    kwargs = dict(region=[0, 3, 0, 5], projection="X3c/5c", frame=0)
+    kwargs = dict(region=(0, 3, 0, 5), projection="X3c/5c", frame=0)
```

Comment:
Any specific reason for changing `region=[0, 3, 0, 5]` to `region=(0, 3, 0, 5)`?

Reply:
Oh not really, I just wanted to highlight the `frame=0` case. Should I revert this change?

Reply:
> Should I revert this change?

Yes.